### ISSUE TITLE
LinkRowCard design updates

### ISSUE THREAD

**Issue by radoslavzeman** [bratislava/bratislava.sk#2065]
What needs to be done:
- truncate url
- use `aria-hidden` on url
- use plain button instead of outlined button

Context:

Url in LinkRowCard can be pretty long in some cases (sharepoint links, google docs etc.) and we should truncate it. It's up to dev decision how many lines will be kept on mobile/desktop.

Moreover, since the card itself _is_ a link, showing url to screen readers is redundant. Therefore, url should be marked as `aria-hidden`.

As suggested from UX designer, we should use plain button instead of outlined button, since outlined button seems very pronounced in this case.

Figma:
There is no design for this section. (https://www.figma.com/design/17wbd0MDQcMW9NbXl6UPs8/DS--Component-library?node-id=19100-18868&m=dev)

Examples:

https://bratislava.sk/vzdelavanie-a-volny-cas/starz/prevadzky-sportoviska/mestska-plavaren-pasienky-50m

<img width="919" height="393" alt="Image" src="https://github.com/user-attachments/assets/9577ea52-a7bb-471e-a7f9-8b2ca1fac8cf" />

https://bratislava.sk/socialne-sluzby-a-byvanie/socialne-sluzby-a-zariadenia/zariadenia-a-sluzby-pre-seniorov/petrzalsky-domov-seniorov

<img width="853" height="214" alt="Image" src="https://github.com/user-attachments/assets/dce1649b-bd9e-4e00-93fa-dd48459d2a13" />
